docs/ipxe_home_server.md

```diff
@@ -0,0 +1,57 @@
+# iPXE on home server #
+
+This guide is about setting up an iPXE-tftp server on home server machine.
+
+## Installation ##
+
+1. Server should be running inside a Docker container therefore docker is
+required. To install docker on Debian go [here](https://docs.docker.com/engine/installation/linux/debian/).
+
+2. To build environment and install ipxe server go to [this](https://github.com/3mdeb/pxe-server)
+address and follow instructions. I belive that according to FHS, pxe-server
```

Comment:
@leszekbulawa I'm not sure if it is important to say what you belive :) Just say `pxe-server should be cloned to`